sovtoken/sovtoken/request_handlers/read_req_handler/get_utxo_handler.py

```diff
@@ -0,0 +1,69 @@
+from sovtoken.constants import ADDRESS, OUTPUTS
+from sovtoken.messages.txn_validator import txt_get_utxo_validate
+from sovtoken.request_handlers.token_utils import parse_state_key
+from sovtoken.types import Output
+from sovtoken.util import SortedItems
+
+from common.serializers.serialization import state_roots_serializer, proof_nodes_serializer
+from plenum.common.constants import MULTI_SIGNATURE, ROOT_HASH, PROOF_NODES, STATE_PROOF
+from plenum.common.exceptions import InvalidClientRequest
+from plenum.common.request import Request
+from plenum.common.types import f
+from plenum.server.database_manager import DatabaseManager
+from plenum.server.request_handlers.handler_interfaces.read_request_handler import ReadRequestHandler
+from state.trie.pruning_trie import rlp_decode
+
+
+class GetUtxoHandler(ReadRequestHandler):
+    def __init__(self, database_manager: DatabaseManager, txn_type, ledger_id, bls_store):
```

Comment:
txn_type == GET_UTXO? Please, don't pass it into constructor for GetUtxoHandler and pass it for ReadRequestHandler